hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/namenode/FSNamesystem.java

```diff
@@ -8838,6 +8838,10 @@ private Supplier<String> getLockReportInfoSupplier(String src, String dst,
       UserGroupInformation ugi = Server.getRemoteUser();
       String userName = ugi != null ? ugi.toString() : null;
       InetAddress addr = Server.getRemoteIp();
+      if (addr != null) {
+        // The host name resolution on InetAddress provides Hostname with IP address
+        addr.getHostName();
```

Comment:
The parsing result is not used, are you sure it is useful?

Reply:
Yes it is useful because once you perform hostname resolution, the implementation of InetAddress#toString lets you print both hostname as well as ip address.